web/templates/submission/index.html

```diff
@@ -177,8 +181,8 @@
                                 {% endif %}
                                 {% if resubmit %}
                                     <div class="form-group">
-                                        <label for="form_package">Task Category</label>
-                                        <select class="form-control" id="form_package" name="job_category">
+                                        <label for="form_job_category">Task Category</label>
+                                        <select class="form-control" id="form_job_category" name="job_category">
```

Comment:
This is a quick bug fix, it looks like the `label for` and `select id` were copy/pasted from the `form_package` down below